### ISSUE TITLE
Miner crashing during sector status change

### ISSUE THREAD

**Issue by funktafide** [filecoin-project/lotus#4466]
**Describe the bug**
Miner seems to crash at night for the last two nights.  Sealing two sectors and 1 sector in WaitDeals.  This is happening when I sleep, so Im not sure if the time coincides with PoSt.  Errors I have in miner.log:

**Update**
This seems to happen during sector status change.  From PC1 to PC2 and others.  

2020-10-18T22:19:19.580+0800	ERROR	fsm	fsm/fsm.go:92	Executing event planner failed: Invalid transition in queue, state `17`, event `13`:
    github.com/filecoin-project/go-statemachine/fsm.eventProcessor.Apply
        /home/funk/go/pkg/mod/github.com/filecoin-project/go-statemachine@v0.0.0-20200925024713-05bd7c71fbfe/fsm/eventprocessor.go:137
2020-10-18T22:19:19.580+0800	ERROR	fsm	fsm/fsm.go:92	Executing event planner failed: Invalid transition in queue, state `17`, event `13`:
    github.com/filecoin-project/go-statemachine/fsm.eventProcessor.Apply
        /home/funk/go/pkg/mod/github.com/filecoin-project/go-statemachine@v0.0.0-20200925024713-05bd7c71fbfe/fsm/eventprocessor.go:137
2020-10-18T22:19:19.580+0800	ERROR	fsm	fsm/fsm.go:92	Executing event planner failed: Invalid transition in queue, state `17`, event `13`:
    github.com/filecoin-project/go-statemachine/fsm.eventProcessor.Apply
        /home/funk/go/pkg/mod/github.com/filecoin-project/go-statemachine@v0.0.0-20200925024713-05bd7c71fbfe/fsm/eventprocessor.go:137
2020-10-18T22:19:19.580+0800	ERROR	fsm	fsm/fsm.go:92	Executing event planner failed: Invalid transition in queue, state `17`, event `13`:
    github.com/filecoin-project/go-statemachine/fsm.eventProcessor.Apply
        /home/funk/go/pkg/mod/github.com/filecoin-project/go-statemachine@v0.0.0-20200925024713-05bd7c71fbfe/fsm/eventprocessor.go:137
2020-10-18T22:19:19.580+0800	ERROR	fsm	fsm/fsm.go:92	Executing event planner failed: Invalid transition in queue, state `17`, event `13`:
    github.com/filecoin-project/go-statemachine/fsm.eventProcessor.Apply
        /home/funk/go/pkg/mod/github.com/filecoin-project/go-statemachine@v0.0.0-20200925024713-05bd7c71fbfe/fsm/eventprocessor.go:137
2020-10-18T22:19:19.581+0800	ERROR	fsm	fsm/fsm.go:92	Executing event planner failed: Invalid transition in queue, state `17`, event `13`:
    github.com/filecoin-project/go-statemachine/fsm.eventProcessor.Apply
        /home/funk/go/pkg/mod/github.com/filecoin-project/go-statemachine@v0.0.0-20200925024713-05bd7c71fbfe/fsm/eventprocessor.go:137
2020-10-18T22:19:19.580+0800	ERROR	fsm	fsm/fsm.go:92	Executing event planner failed: Invalid transition in queue, state `17`, event `13`:
    github.com/filecoin-project/go-statemachine/fsm.eventProcessor.Apply
        /home/funk/go/pkg/mod/github.com/filecoin-project/go-statemachine@v0.0.0-20200925024713-05bd7c71fbfe/fsm/eventprocessor.go:137
2020-10-18T22:19:19.581+0800	ERROR	fsm	fsm/fsm.go:92	Executing event planner failed: Invalid transition in queue, state `17`, event `13`:
    github.com/filecoin-project/go-statemachine/fsm.eventProcessor.Apply
        /home/funk/go/pkg/mod/github.com/filecoin-project/go-statemachine@v0.0.0-20200925024713-05bd7c71fbfe/fsm/eventprocessor.go:137
2020-10-18T22:19:19.581+0800	ERROR	fsm	fsm/fsm.go:92	Executing event planner failed: Invalid transition in queue, state `17`, event `13`:
    github.com/filecoin-project/go-statemachine/fsm.eventProcessor.Apply
        /home/funk/go/pkg/mod/github.com/filecoin-project/go-statemachine@v0.0.0-20200925024713-05bd7c71fbfe/fsm/eventprocessor.go:137
2020-10-18T22:19:19.581+0800	ERROR	fsm	fsm/fsm.go:92	Executing event planner failed: Invalid transition in queue, state `17`, event `13`:
    github.com/filecoin-project/go-statemachine/fsm.eventProcessor.Apply
        /home/funk/go/pkg/mod/github.com/filecoin-project/go-statemachine@v0.0.0-20200925024713-05bd7c71fbfe/fsm/eventprocessor.go:137
2020-10-18T22:19:19.581+0800	ERROR	fsm	fsm/fsm.go:92	Executing event planner failed: Invalid transition in queue, state `17`, event `13`:
    github.com/filecoin-project/go-statemachine/fsm.eventProcessor.Apply
        /home/funk/go/pkg/mod/github.com/filecoin-project/go-statemachine@v0.0.0-20200925024713-05bd7c71fbfe/fsm/eventprocessor.go:137
2020-10-18T22:19:19.581+0800	ERROR	fsm	fsm/fsm.go:92	Executing event planner failed: Invalid transition in queue, state `17`, event `13`:
    github.com/filecoin-project/go-statemachine/fsm.eventProcessor.Apply
        /home/funk/go/pkg/mod/github.com/filecoin-project/go-statemachine@v0.0.0-20200925024713-05bd7c71fbfe/fsm/eventprocessor.go:137
2020-10-18T22:19:19.581+0800	ERROR	fsm	fsm/fsm.go:92	Executing event planner failed: Invalid transition in queue, state `17`, event `13`:
    github.com/filecoin-project/go-statemachine/fsm.eventProcessor.Apply
        /home/funk/go/pkg/mod/github.com/filecoin-project/go-statemachine@v0.0.0-20200925024713-05bd7c71fbfe/fsm/eventprocessor.go:137
2020-10-18T22:19:19.581+0800	ERROR	fsm	fsm/fsm.go:92	Executing event planner failed: Invalid transition in queue, state `17`, event `13`:
    github.com/filecoin-project/go-statemachine/fsm.eventProcessor.Apply
        /home/funk/go/pkg/mod/github.com/filecoin-project/go-statemachine@v0.0.0-20200925024713-05bd7c71fbfe/fsm/eventprocessor.go:137
2020-10-18T22:19:19.581+0800	ERROR	fsm	fsm/fsm.go:92	Executing event planner failed: Invalid transition in queue, state `17`, event `13`:
    github.com/filecoin-project/go-statemachine/fsm.eventProcessor.Apply
        /home/funk/go/pkg/mod/github.com/filecoin-project/go-statemachine@v0.0.0-20200925024713-05bd7c71fbfe/fsm/eventprocessor.go:137
2020-10-18T22:19:19.581+0800	ERROR	fsm	fsm/fsm.go:92	Executing event planner failed: Invalid transition in queue, state `17`, event `13`:
    github.com/filecoin-project/go-statemachine/fsm.eventProcessor.Apply
        /home/funk/go/pkg/mod/github.com/filecoin-project/go-statemachine@v0.0.0-20200925024713-05bd7c71fbfe/fsm/eventprocessor.go:137
2020-10-18T22:19:19.581+0800	ERROR	fsm	fsm/fsm.go:92	Executing event planner failed: Invalid transition in queue, state `17`, event `13`:
    github.com/filecoin-project/go-statemachine/fsm.eventProcessor.Apply
        /home/funk/go/pkg/mod/github.com/filecoin-project/go-statemachine@v0.0.0-20200925024713-05bd7c71fbfe/fsm/eventprocessor.go:137
2020-10-18T22:19:19.581+0800	ERROR	fsm	fsm/fsm.go:92	Executing event planner failed: Invalid transition in queue, state `17`, event `13`:
    github.com/filecoin-project/go-statemachine/fsm.eventProcessor.Apply
        /home/funk/go/pkg/mod/github.com/filecoin-project/go-statemachine@v0.0.0-20200925024713-05bd7c71fbfe/fsm/eventprocessor.go:137
2020-10-18T22:19:19.581+0800	ERROR	fsm	fsm/fsm.go:92	Executing event planner failed: Invalid transition in queue, state `17`, event `13`:
    github.com/filecoin-project/go-statemachine/fsm.eventProcessor.Apply
        /home/funk/go/pkg/mod/github.com/filecoin-project/go-statemachine@v0.0.0-20200925024713-05bd7c71fbfe/fsm/eventprocessor.go:137
2020-10-18T22:19:30.454+0800	INFO	rpc	go-jsonrpc@v0.1.2-0.20201008195726-68c6a2704e49/client.go:346	rpc output message buffer	{"n": 2}
2020-10-18T22:19:30.456+0800	INFO	rpc	go-jsonrpc@v0.1.2-0.20201008195726-68c6a2704e49/client.go:346	rpc output message buffer	{"n": 2}
2020-10-18T22:20:00.581+0800	INFO	rpc	go-jsonrpc@v0.1.2-0.20201008195726-68c6a2704e49/client.go:346	rpc output message buffer	{"n": 2}
2020-10-18T22:20:00.825+0800	INFO	rpc	go-jsonrpc@v0.1.2-0.20201008195726-68c6a2704e49/client.go:346	rpc output message buffer	{"n": 3}
2020-10-18T22:20:33.027+0800	INFO	rpc	go-jsonrpc@v0.1.2-0.20201008195726-68c6a2704e49/client.go:346	rpc output message buffer	{"n": 4}
2020-10-18T22:21:10.182+0800	INFO	rpc	go-jsonrpc@v0.1.2-0.20201008195726-68c6a2704e49/client.go:346	rpc output message buffer	{"n": 5}
2020-10-18T22:22:14.900+0800	INFO	graphsync	impl/graphsync.go:260	Graphsync ReceiveError: stream reset

**To Reproduce**
1. Runing lotus-miner
2. Sealing 2 sectors with deals
3. 1 sector in WaitDeals

**Expected behavior**
Expected miner to continue sealing two sectors then start sealing sector in WaitDeals.

**Version (run `lotus version`):**
Daemon:  0.10.2+git.8bb84acb+api0.17.0
Local: lotus version 0.10.2+git.8bb84acb

**Additional context**
Unsure what the issue is. First day the problem happened PoSt was successful. Second day PoSt failed.

**Comment by coinmini** [comment]
same problem here with  windowPoSt failed

**Comment by NSC-FIL** [comment]
On lotus version 1.1.0+git.420ec18c5 , I get the same error intermittently.  Seems to happen when two sectors, where at least one has deals, are sealing in parallel.

**Comment by TippyFlitsUK** [comment]
Also experiencing this error in v1.1.2.

`lotus-miner version 1.1.2+git.d4cdc6d3`

**Comment by funktafide** [comment]
Im still having this problem on v1.1.2 as well.  Seems to be in line with what NSC-FIL is saying.  I was pledging fine with empty sectors.  I had a deal in one today and one without and it crashed.

**Comment by rjan90** [comment]
I think this issue been resolved on the newer versions of Lotus? Can this be closed now? #rengjøring

**Comment by jennijuju** [comment]
Please open a new ticket if you are running into this issue on the latest version of lotus!
